Makefile

```diff
@@ -11,14 +11,24 @@ clean: ## Delete pyc files
 build: ## Build Docker image
 	@docker build -t "${DOCKER_TAG}" .
 
-.PHONY: tests
-tests: clean build ## Run tests in container
+.PHONY: docker-tests
+docker-ui-tests: clean build ## Run tests in container
 	@docker run \
 		--net="host" \
 		--env REDASH_SERVER_URL="${REDASH_SERVER_URL}" \
 		--mount type=bind,source="${CURDIR}",target=/home/user/src \
 		"${DOCKER_TAG}"
 
+.PHONY: ui-tests
+ui-tests: ## Run tests outside of container
+	@pipenv run pytest --driver=Firefox --server-url=http://localhost:5000 --verify-server-url --variables=variables.json --html=report.html
```

Comment:
Do you think we still need the docker command? We could point just have an entrypoint.

Reply:
I think it makes sense to have an entrypoint of

```
ENTRYPOINT [ "pipenv", "run" ]
```

so that all commands run inside of our virtual environment.

The command should be ``pytest`` by-default and can be overwritten if needed.

Reply:
But with this we couldn't run anything else besides that one command.